### PR TITLE
Remove local rule

### PR DESCRIPTION
**Pull request by askoufis**
`eslint-plugin-local-rule` does a `require` up the file tree to find local rules in specific files/folders. If this package is hoisted (as is potentially the case in yarn v1), it will never find the local rules inside `eslint-config-seek`. For this reason, I'm removing the rule as it doesn't really have a huge impact on most cypress tests, and it's not worth breaking all yarn consumers.

Not marking this change as breaking as the rule never worked in the first place.

### CODE REVIEW

**Review by jahredhope — APPROVED** [review]

**Review by georgespyropoulos — APPROVED** [review]

File: .changeset/eight-hounds-develop.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+'eslint-config-seek': patch
+---
+
+Remove local eslint rule added in [`v11.1.2`]
+
+The local `unsafe-to-chain-command` cypress rule required `eslint-plugin-local-rules` to function, but that plugin doesn't work correctly when yarn hoists it to the top of `node_modules`.
+The application of the rule isn't particularly common, and it's not worth breaking yarn consumers, so removing it for now until either the fork or the upstream `eslint-plugin-cypress` supports it.
```
Comment (mrm007) [inline .changeset/eight-hounds-develop.md:8]: > The application of the rule isn't particularly common

Looks like there is [one consumer](https://github.com/search?q=org:SEEK-Jobs%20unsafe-to-chain-command&type=code) that uses the mentioned rule. Have you considered https://github.com/not-an-aardvark/eslint-plugin-rulesdir as an alternative to `eslint-plugin-local-rules`?

Comment (askoufis) [reply .changeset/eight-hounds-develop.md:8]: That plugin looks like it'll solve the issue since I can resolve the package explicitly.

**Comment by askoufis** [comment]
Looking into https://github.com/not-an-aardvark/eslint-plugin-rulesdir. Will either close this PR and open a new one if it works, or just merge this one if not.

**Comment by askoufis** [comment]
Closing in favour of #111.

**Pull request closed without merging**